pkg/fileutils/read.go

```diff
@@ -95,6 +55,17 @@ func readAbsFile(file string) ([]byte, error) {
 	return ioutil.ReadFile(file)
 }
 
+func fileURLPath(u *url.URL) string {
+	file := u.Path
+
+	// Sanitise path on Windows. See https://github.com/golang/go/issues/6027
+	if runtime.GOOS == "windows" && file[0] == '/' {
```

Comment:
Can't `file` be the empty string?

Reply:
I don't _think_ file can be empty because, currently, the callers create the input URL by parsing a string and I don't think that can result in an empty path. However, I agree that the test isn't robust since a URL path could in general be empty, so I'll replace the condition with:
```
runtime.GOOS == "windows" && len(file) > 0 && file[0] == '/'
```